src/main/java/seedu/duke/command/FindCommand.java

```diff
@@ -16,10 +15,19 @@
 
     private static final String COMMAND_USAGE = COMMAND_WORD + ": Finds a note. Parameters: KEYWORDS";
 
+    /**
+     * Gets how the command is expected to be used.
+     *
+     * @return String representation of how the command is to be used.
+     */
     public static String getCommandUsage() {
         return COMMAND_USAGE;
     }
 
+    private static final String COMMAND_UNSUCCESSFUL_MESSAGE = "There are no matching notes. Please try another search "
+            + "query.";
```

Comment:
I feel this breaks the flow.
```suggestion
    private static final String COMMAND_UNSUCCESSFUL_MESSAGE = "There are no matching notes. "
            + "Please try another search query.";
```